src/backend/marsha/core/utils/xmpp_utils.py

```diff
@@ -54,6 +54,51 @@ def create_room(room_name):
         )
     )
 
+    # request the default config when a room is created
+    default_config_iq = client.SendAndWaitForResponse(
+        xmpp.Iq(
+            to=f"{room_name}@{settings.XMPP_CONFERENCE_DOMAIN}",
+            frm=settings.XMPP_PRIVATE_ADMIN_JID,
+            typ="get",
+            queryNS=xmpp.NS_MUC_OWNER,
+        )
+    )
+
+    data = []
+    fileds_to_exclude = [
+        "muc#roomconfig_persistentroom",
+        "muc#roomconfig_publicroom",
+        "muc#roomconfig_allowpm",
+        "muc#roomconfig_allowinvites",
+        "muc#roomconfig_changesubject",
+    ]
+
+    # Remove config we want to modify
+    for children in default_config_iq.getQueryPayload()[0].getChildren():
+        if (
+            children.getName() == "field"
+            and children.getAttr("var") not in fileds_to_exclude
+        ):
+            data.append(children)
+
+    # Add our own config
+    data = data + [
+        # Make Room Persistent?
```

Comment:
why did you add question tag with your comment

Reply:
It was to describe the form field but yes I will remove the question mark.